released/packages/coq-vellvm/coq-vellvm.1.0.20240416/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+name: "coq-vellvm"
+maintainer: "stevez@cis.upenn.edu"
+synopsis: "Coq library implementing (executable) semantics for LLVM IR"
+
+homepage: "https://github.com/vellvm/vellvm"
+dev-repo: "git+https://github.com/vellvm/vellvm.git"
+bug-reports: "https://github.com/vellvm/vellvm/issues"
+authors: [
+  "Steve Zdancewic <stevez@cis.upenn.edu>"
+  "Yannick Zakowski <yannick.zakowski@inria.fr>"
+  "Calvin Beck <hobbes@seas.upenn.edu>"
+  "Irene Yoon <euisuny@cis.upenn.edu>"
+  "Gary (Hanxi) Chen <hanxic@seas.upenn.edu>"
+]
+license: "GPL-3.0-or-later"
+
+
+build: [make "-C" "src" "all" "-j%{jobs}%"]
+install: [make "-C" "src" "install"]
+    
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "cppo" 
+  "dune" 
+  "menhir" 
+  "qcheck" 
+  "coq" {>= "8.19.0" & < "8.20~"}
+  "coq-ext-lib" {< "0.12.1"}
+  "coq-paco" 
+  "coq-ceres" 
+  "coq-flocq" {>= "4.1.1"}
+  "coq-mathcomp-ssreflect" 
+  "coq-simple-io"
+  "coq-itree" {>= "5.1.2" & < "5.2~"}
+  "coq-quickchick" {>= "2.0.2" & < "2.0.3"}
+]
+
+tags: [
+  "date:2023-03-04"
+
+  "category:Computer Science/Programming Languages/Formal Definitions and Theory"
+  "category:Computer Science/Semantics and Compilation/Compilation"
+  "category:Computer Science/Semantics and Compilation/Semantics"  
+
+  "keyword:semantics"
+  "keyword:interpreter"
+  "keyword:LLVM"
+
+  "logpath:Vellvm"
+]
+
+url {
+  src: "https://github.com/vellvm/vellvm/archive/1.0.20240416.tar.gz"
```

Comment:
Since GitHub no longer guarantees stability of zip files and tarballs, we recommend uploading a release archive into the release manually, as is done for Coq: https://github.com/coq/coq/releases/tag/V8.19.1

This is not a requirement, however.

Reply:
Ah, okay, I can do that instead :). I'll get on that right now.

Reply:
@palmskog okay, just trying to understand... I made the release:

https://github.com/vellvm/vellvm/releases/tag/v1.0.20240416

And then I'll refer to it with this source url in the opam file: https://github.com/vellvm/vellvm/releases/tag/v1.0.20240416 ? That's what we want, right?

Reply:
@Chobbes no, what I mean is that you should use GitHub's feature to upload an arbitrary file into a GitHub release. There, you can upload `1.0.20240416.tar.gz`, and it will then get an URL along the lines of https://github.com/vellvm/vellvm/releases/download/v1.0.20240416/1.0.20240416.tar.gz

Compare:
- https://github.com/coq/coq/releases/download/V8.19.1/coq-8.19.1.tar.gz (manually uploaded tarball by Coq release manager, stable over time)
- https://github.com/coq/coq/archive/refs/tags/V8.19.1.tar.gz (GitHub autogenerated tarball, may change checksum at any time)